web/src/css/tailwind.css

```diff
@@ -14,22 +14,28 @@
   }
 }
 
-.btn-normal {
-  @apply select-none inline-flex border cursor-pointer px-3 text-sm leading-8 rounded-md hover:opacity-80 hover:shadow;
-}
-
-.btn-primary {
-  @apply btn-normal border-transparent bg-green-600 text-white;
-}
-
-.btn-danger {
-  @apply btn-normal border-red-600 bg-red-50 text-red-600;
-}
-
-.btn-text {
-  @apply btn-normal text-gray-600 border-none hover:shadow-none;
-}
+@layer components {
+  .btn-normal {
+    @apply select-none inline-flex border cursor-pointer px-3 text-sm leading-8 rounded-md hover:opacity-80 hover:shadow;
+  }
+  
+  .btn-primary {
+    @apply btn-normal border-transparent bg-green-600 text-white;
+  }
+  
+  .btn-danger {
+    @apply btn-normal border-red-600 bg-red-50 text-red-600;
+  }
+  
+  .btn-text {
+    @apply btn-normal text-gray-600 border-none hover:shadow-none;
+  }
+  
+  .input-text {
+    @apply w-full px-3 py-2 leading-6 text-sm border rounded;
+  }
 
-.input-text {
-  @apply w-full px-3 py-2 leading-6 text-sm border rounded;
+  .card-memo {
```

Comment:
As it's used in only two places, it doesn't look like a common util class. 

https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction

Reply:
But it looks like both need to be styled the same way, and also the cards in the home page are styled the same

Reply:
We can create a less/module.css file including the same layout css codes. Then import it in the component files. 

The tailwind.css should save the basic class definitions which will be used in multiple files.